index.html

```diff
@@ -0,0 +1,184 @@
+          <h2 class="text3">2022.3.21(MON) ~ 25 (FRI) San Francisco, CA</h2>
+        </div>
+      </div>
+    </section>
+    <section class="second-section">
+      <div class="second-content">
+        <h2 class="text4">Main Program</h2>
+        <div class="line"></div>
+      </div>
+      <div class="flexbox">
+        <div class="item">
+          <img
+            src="images/lecturer.png"
+            class="item-img margin0"
+            alt="lecturer"
+          />
+          <h2 class="item-title margin0">Lecture</h2>
+          <p class="item-text light tex-desk">
+            Listen to the speakers from various countries about the messages of
+            sharing and opening.
+          </p>
+        </div>
+        <div class="item">
+          <img
+            src="images/painting.png"
+            class="item-img margin0"
+            alt="Exhibition"
+          />
+          <h2 class="item-title margin0">Exhibition</h2>
+          <p class="item-text light tex-desk">
+            Appreciate various creations, organized from the GDC.
+          </p>
+        </div>
+        <div class="item">
+          <img src="images/faq.png" class="item-img margin0" alt="faq" />
+          <h2 class="item-title margin0">Forum</h2>
+          <p class="item-text light tex-desk">
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </div>
+        <div class="item">
+          <img
+            src="images/workshop.png"
+            class="item-img margin0"
+            alt="workshop"
+          />
+          <h2 class="item-title margin0">Workshop</h2>
+          <p class="item-text light tex-desk">
+            Try creating your own work using open source license rather than
+            just watching.
+          </p>
+        </div>
+        <div class="item">
+          <img
+            src="images/network.png"
+            class="item-img margin0"
+            alt="network"
+          />
+          <h2 class="item-title margin0">Ignite</h2>
+          <p class="item-text light tex-desk">
+            Get opportunities to network with people around the worlk, also
+            after the summit.
+          </p>
+        </div>
+      </div>
+      <a class="program" href="#">SEE THE WHOLE PROGRAM</a>
+      <a class="btn" href="#">Join GDC Global summit 2022</a>
+    </section>
+    <section class="third-section">
+      <h2 class="text5">Featured Speakers</h2>
+      <div class="line2"></div>
+      <div class="speakers"></div>
+    </section>
+    <section class="partner">
+      <h2 class="text4">Partner</h2>
+      <div class="line2"></div>
+      <div class="partner-flex">
+        <img class="partner-icon" src="images/mozella.png" alt="mozella-icon" />
+        <img class="partner-icon" src="images/google.png" alt="google-icon" />
+        <img class="partner-icon" src="images/airbnb.png" alt="airbnb-icon" />
+      </div>
+    </section>
```

Comment:
- Your Partners' section looks amazing. 💯 However, I think it'll look more awesome when you add the other partners just as the template shows. 😉

- Your Design 👇 

![](https://i.imgur.com/OG6rPkR.png)

- Original Design 👇 

![](https://i.imgur.com/EHX5KAf.png)